README.md

```diff
@@ -278,7 +278,8 @@ If you need to read from the state to change it, you should do this inside the a
 
 ## Licenses and Attributions
 
--   License information about used images can be found [here](frontend/src/assets/image-sources.md). All images are licensed under their original license.
+-   Read [LICENSE-README.md](LICENSE-README.md) for information about the used license
+-   License information about used images can be found [here](frontend/src/assets/image-sources.md). All images are licensed under their original license, for further information what is meant with an image, look at [LICENSE-README.md](LICENSE-README.md).
```

Comment:
```suggestion
Digital Fuesim MANV
Copyright (C) 2023 HPI-SAM

This program is free software: you can redistribute it and/or modify
it under the terms of the GNU Affero General Public License as
published by the Free Software Foundation, either version 3 of the
License, or (at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU Affero General Public License for more details.

You should have received a copy of the GNU Affero General Public License
along with this program.  If not, see <https://www.gnu.org/licenses/>.

License information about used images can be found in [frontend/src/assets/image-sources.md](frontend/src/assets/image-sources.md). All images are licensed under their original license, for further about licensing of images and datasets refer to [LICENSE-README.md](LICENSE-README.md).
```

Reply:
@Greenscreen23 Are you ok with the text added? I am not sure about the text with images and datasets.

Reply:
Its probably fine